Formula/kubeless.rb

```diff
@@ -0,0 +1,36 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless.git",
+      :tag => "v0.2.3",
+      :revision => "bfefb404e22a209659ef06b98a51051d72dd5c21"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :recommended
+
+  def install
+    commit = Utils.popen_read("git rev-parse --short HEAD").chomp
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/kubeless/kubeless").install Dir["*"]
+    cd "src/github.com/kubeless/kubeless" do
+      ldflags = ["-w -X github.com/kubeless/kubeless/version.VERSION=v#{version}",
```

Comment:
It's not a huge thing but it might be worth using a `%W[]` style array here. It'll let you drop the commas and quotes and it just looks a bit neater.